shell/edit/auth/oidc.vue

```diff
@@ -107,7 +115,17 @@ export default {
      */
     requiredScopes() {
       return this.model.id ? (BASE_SCOPES[this.model.id] || []) ? (BASE_SCOPES[this.model.id] || [])[0].split(' ') : [] : [];
-    }
+    },
+
+    requiresCert() {
+      // We assume all do, apart from the ones here, which do not
+      return !(['cognito'].includes(this.model.id));
```

Comment:
optional, given we haven't done it elsewhere, but there's a lot more references now...

should we start a process to have auth provider names typed somewhere? const / enum, just something to avoid the strings everywhere

Reply:
Yeah - we should. I'm not going to do it this time though! I think we should have clearly configuration for auth providers, so you'd declare cognito and indicate if it needed certs etc - perhaps we'd have `OIDCAuthProviderConfig` for OIDC types etc.